queue-worker/main.go

```diff
@@ -48,150 +49,163 @@ func makeClient() http.Client {
 	return proxyClient
 }
 
-func main() {
-	log.SetFlags(0)
+const (
+	subj      = "faas-request"
+	qgroup    = "faas"
+	clusterID = "faas-cluster"
+)
 
-	clusterID := "faas-cluster"
-	val, _ := os.Hostname()
-	clientID := "faas-worker-" + val
+var (
```

Comment:
I would prefer refactoring for style to be done in subsequent PRs as it blurs the intent of the PR and makes it harder to review the change.